frontend/src/components/pages/student/AssessmentSummaryPage.tsx

```diff
@@ -29,29 +29,41 @@ const AssessmentSummaryPage = (): React.ReactElement => {
 
   return (
     <>
-      {test && (
+      {test && testSession && (
```

Comment:
I don't know how I feel about this b/c technically this page would never show if these fields were null... (b/c that case is handled in `StudentDashboard` ), but idk how to make this better lol

Reply:
hmm its fine lol. here's a [thread](https://stackoverflow.com/questions/67785257/best-practice-when-asserting-non-null-on-a-value-coming-from-react-usecontext-o) of ideas but i feel like it's kinda extra